swap/types.go

```diff
@@ -52,5 +55,5 @@ type ErrorMsg struct{}
 
 // ConfirmMsg is sent from the creditor to the debitor to confirm cheque reception
 type ConfirmMsg struct {
-	Cheque Cheque // TODO: probably not needed and if so likely should not include the full cheque
+	Cheque *Cheque // TODO: probably not needed and if so likely should not include the full cheque
```

Comment:
This results in error `Invalid message (RLP error): <= msg #3 (6 bytes): rlp: too few elements for swap.Cheque, decoding into (swap.ConfirmMsg).Cheque` when the `ConfirmMsg` is received (probably because it is sent with `sp.Send(ctx, &ConfirmMsg{})`). For some reason this doesn't result in an error if it's not a pointer.